content/en/blog/grpconf-2024-videos.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Highlights in gRPConf 2024:Customer Showcase, Developer Engagement, Birds of Feathers Discussions and more.
+date: 2024-07-03
```

Comment:
why this date? change to the date that this blog post is published?

Reply:
done